code/modules/recycling/sortingmachinery.dm

```diff
@@ -8,18 +8,24 @@
 	var/giftwrapped = FALSE
 	var/sortTag = 0
 	var/obj/item/paper/note
+	var/obj/item/barcode/sticker
 
 /obj/structure/bigDelivery/interact(mob/user)
 	playsound(src.loc, 'sound/items/poster_ripped.ogg', 50, TRUE)
+	new /obj/effect/decal/cleanable/wrapping(get_turf(user))
 	qdel(src)
 
 /obj/structure/bigDelivery/Destroy()
 	var/turf/T = get_turf(src)
+	for(var/obj/I in src.GetAllContents())
+		SEND_SIGNAL(I, COMSIG_ITEM_UNWRAPPED)
 	for(var/atom/movable/AM in contents)
 		AM.forceMove(T)
 	return ..()
 
 /obj/structure/bigDelivery/contents_explosion(severity, target)
+	for(var/obj/I in src.GetAllContents())
+		SEND_SIGNAL(I, COMSIG_ITEM_UNWRAPPED)

```

Comment:
Same here, it isn't being unwrapped so what's this doing here?

Reply:
This one is my bad, I was under the impression that "Contents_explosion" meant "All the things leaving the wrapped crate".